opam-client.opam

```
@@ -1,5 +1,5 @@
 opam-version: "2.0"
-version: "2.3.0~alpha2~dev"
+version: "2.3.0~beta1"
 synopsis: "Client library for opam 2.2"
```

```suggestion
synopsis: "Client library for opam 2.3"
```
We missed that after 2.2, maybe to add in release process file? or just remove the version?